rust/tools/authorization-logic/src/main.rs

```diff
@@ -31,22 +31,21 @@ use structopt::StructOpt;
 #[derive(StructOpt)]
 #[structopt(name = "auth-logic", about = "An authorization logic compiler.")]
 struct Opt {
-    /// The name of the auth logic program to compile
+    /// The path of the input auth logic program to compile
     // First positional argument (should be passed as a raw value, not --filename=...)
     #[structopt(required = true)]
-    filename: String,
+    input_filename: String,
 
-    /// Input directory where the auth logic program lives.
-    /// If unspecified, it will be the current working directory.
-    // Can be passed as -i or --in-dir
-    #[structopt(short, long, env("PWD"))]
-    in_dir: String,
+    /// Output file path where compiled souffle program will be saved.
+    // Can be passed as -o or --souffle_output_file
+    #[structopt(short = "o", long)]
+    souffle_output_file: String,
 
-    /// Output directory, where the compiled datalog program will be saved.
-    /// If unspecified, it will be the current working directory.
+    /// Output directory, where the CSVs containing the results of evaluating
+    /// queries will be saved.
     // Can be passed as -o or --out-dir
```

Comment:
I see some `//` and some `///`. Is that intentional?

Reply:
Yes. The `///` is meant to be included in the doc comments whereas `//` is not.